src/app/pages/apps/metering-profile/metering-profile-routing.module.ts

```diff
@@ -0,0 +1,39 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { QuicklinkModule } from 'ngx-quicklink';
+import { VexRoutes } from '../../../../@vex/interfaces/vex-route.interface';
+import { MeteringProfileListComponent } from './metering-profile-list/metering-profile-list.component';
+import { MeteringProfileComponent } from './metering-profile.component';
+import { PageMode } from 'src/app/core/enums/page-mode.enum';
+
+const routes: VexRoutes = [
+  {
+    path: '',
+    component: MeteringProfileListComponent,
+    data: {
+      toolbarShadowEnabled: false
+    }
+  },
+  {
+    path: 'edit/:id',
+    component: MeteringProfileComponent,
+    data: {
+      pageMode: PageMode.Edit,
+      toolbarShadowEnabled: false
+    }
+  },
+  {
+    path: 'add',
+    component: MeteringProfileComponent,
+    data: {
```

Comment:
we can add pageMode here as well to set as standard

Reply:
applied.